packages/nextjs/src/app-router/server/auth.ts

```diff
@@ -51,6 +50,25 @@ export const auth = (): Auth => {
   return Object.assign(authObject, { protect, redirectToSignIn });
 };
 
+export const auth = (): Auth | Record<string, any> => {
+  require('server-only');
+
+  if (!EPHEMERAL_MODE_AVAILABLE) {
+    return baseAuth();
+  }
+
+  // NOTE: This is a workaround to allow the development environment to use the clerk key
+  // without having to set the environment variables.
+  try {
+    return baseAuth();
+  } catch (e: any) {
+    if (isClerkKeyError(e)) {
+      return initialState();
+    }
+    throw e;
+  }
+};
+
```

Comment:
Wonder if we may be able to remove this if we use the `clerkMiddlewareRequestDataStore` wrapper - need to test once main is merged in.